### ISSUE TITLE
Incorrect max value defined for volume controller.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1352]
**Actual Result:**
Incorrect max value defined for volume controller.

**Note** - Same issue repro on where video and audio clips are present.

**Expected Result:**
Correct max value should be defined for volume controller.

**Repro Steps:**
1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Start Narrator by pressing "Ctrl + Win + Enter" key.
3. Navigate to 'Type your message' text box present in bottom of screen, type 'help' and press enter key.
4. Navigate to 'Show an audio attachment' button and activate it.
5. Navigate to volume controller present on the music bar on the bottom of the page.
6. Observe narrator's announcement on the volume controller.

**User Impact:**
If incorrect max value is defined for volume controller then the user who is dependent on screen reader might get confuse about the range of that controller.

**Test Environment:**
OS: Windows 10
OS Build: 17763.107
OS Version: 1809
Browser: Edge

**Attachment:**
[Usable_IncorrectMaxValue.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2580689/Usable_IncorrectMaxValue.zip)

**Comment by corinagum** [comment]
Confirmed as external with accessibility team

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.1.2
